<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск вещи</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="color_label">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="color_box">
       <item>
        <property name="text">
         <string>Любой</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="result_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>731</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>124</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
